src/overlays/actors/ovl_Magic_Dark/z_magic_dark.c

```diff
@@ -12,14 +12,13 @@
 
 void MagicDark_Init(Actor* thisx, GlobalContext* globalCtx);
 void MagicDark_Destroy(Actor* thisx, GlobalContext* globalCtx);
-void MagicDark_Update(Actor* thisx, GlobalContext* globalCtx);
-void MagicDark_Draw(Actor* thisx, GlobalContext* globalCtx);
+void MagicDark_OrbUpdate(Actor* thisx, GlobalContext* globalCtx);
+void MagicDark_OrbDraw(Actor* thisx, GlobalContext* globalCtx);
 
-void func_80B874E4(Actor* thisx, GlobalContext* globalCtx);
-void func_80B8772C(GlobalContext* globalCtx, f32 a1);
-void func_80B87A18(Actor* thisx, GlobalContext* globalCtx);
+void MagicDark_DiamondUpdate(Actor* thisx, GlobalContext* globalCtx);
+void MagicDark_DimLighting(GlobalContext* globalCtx, f32 intensity);
+void MagicDark_DiamondDraw(Actor* thisx, GlobalContext* globalCtx);
```

Comment:
I suggest grouping "main" functions together (init/destroy/update/draw) and leaving others like DimLighting apart.